README.md

```diff
@@ -1003,45 +1026,40 @@ DownscopedCredentials downscopedCredentials =
 AccessToken downscopedAccessToken = downscopedCredentials.refreshAccessToken();
 ```
 
-A token broker can be set up on a server in a private network. Various workloads 
-(token consumers) in the same network will send authenticated requests to that broker for downscoped
-tokens to access or modify specific google cloud storage buckets.
+#### Client-side CAB
 
-The broker will instantiate downscoped credentials instances that can be used to generate short 
-lived downscoped access tokens which will be passed to the token consumer. 
+For client-side CAB, the `ClientSideCredentialAccessBoundaryFactory` is used
+with a source credential. After initializing the factory, the `generateToken()`
+method can be called repeatedly with different `CredentialAccessBoundary`
+objects to create multiple downscoped tokens.
 
-Putting it all together:
 ```java
 // Retrieve the source credentials from ADC.
 GoogleCredentials sourceCredentials = GoogleCredentials.getApplicationDefault()
         .createScoped("https://www.googleapis.com/auth/cloud-platform");
 
-// Create an Access Boundary Rule which will restrict the downscoped token to having readonly
-// access to objects starting with "customer-a" in bucket "bucket-123".
```

Comment:
Can you just stick the portion of this code sample that creates the accessboundary rule into the samples for both the client side and server side sections? I think its good to have a single block of code that includes the whole flow.